src/lib/Zikula/Bundle/CoreInstallerBundle/Controller/AjaxInstallController.php

```diff
@@ -337,8 +337,8 @@ private function decodeParameters($params)
     public function reloadRoutes()
     {
         // fire MODULE_INSTALL event to reload all routes
-        $event = new ModuleStateEvent($this->container->get('kernel')->getModule('ZikulaRoutesModule'));
-        $this->container->get('event_dispatcher')->dispatch(CoreEvents::MODULE_POSTINSTALL, $event);
+//        $event = new ModuleStateEvent($this->container->get('kernel')->getModule('ZikulaRoutesModule'));
+//        $this->container->get('event_dispatcher')->dispatch(CoreEvents::MODULE_POSTINSTALL, $event);
```

Comment:
I guess this got disabled because now only customised routes are persisted in the database? If yes, you can also remove it.